src/hotspot/cpu/riscv/c1_Runtime1_riscv.cpp

```diff
@@ -883,7 +883,12 @@ OopMapSet* Runtime1::generate_code_for(C1StubId id, StubAssembler* sasm) {
         __ ld(x10, Address(sp, (sup_k_off) * VMRegImpl::stack_slot_size)); // super klass
 
         Label miss;
-        __ check_klass_subtype_slow_path(x14, x10, x12, x15, nullptr, &miss);
+        __ check_klass_subtype_slow_path(x14,     /*sub_klass*/
+                                         x10,     /*super_klass*/
+                                         x12,     /*tmp1_reg*/
+                                         x15,     /*tmp2_reg*/
+                                         nullptr, /*L_success*/
+                                         &miss    /*L_failure*/);
 
```

Comment:
Could you please add a code comment about extra regsiters used for table lookup? We reserve several more in `MacroAssembler::check_klass_subtype_slow_path_table`.

Reply:
Thanks for your review, and all comments related have been fixed!